ingestors/kafka/src/test/scala/hydra/kafka/utils/FakeV2TopicMetadata.scala

```diff
@@ -0,0 +1,92 @@
+package hydra.kafka.utils
+
+import cats.data.NonEmptyList
+import cats.effect.IO
+import cats.implicits._
+import hydra.avro.registry.SchemaRegistry
+import hydra.avro.registry.SchemaRegistry.SchemaId
+import hydra.kafka.algebras.MetadataAlgebra.TopicMetadataContainer
+import hydra.kafka.algebras.TestMetadataAlgebra
+import hydra.kafka.model.ContactMethod.Email
+import hydra.kafka.model.TopicMetadataV2Request.Subject
+import hydra.kafka.model._
+import org.apache.avro.{Schema, SchemaBuilder}
+
+import java.time.Instant
+
+object FakeV2TopicMetadata {
+
+  private def buildSchema(topic: String, upgrade: Boolean): Schema = {
+    val schemaStart = SchemaBuilder
+      .record("name" + topic.replace("-", "").replace(".", ""))
+      .fields()
+      .requiredString("id" + topic.replace("-", "").replace(".", ""))
```

Comment:
Why do we need this logic with "replace"s?

Reply:
Ok. Removing buildSchema() entirely as this is not required. We had copied this from hydra.ingest.programs.TopicDeletionProgramSpec.buildSchema() where this logic was used.